packages/appstore/src/components/main-title-bar/total-asset.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
@shayan-deriv better to sort the imports with below priority.

external libraries (react, ...)
internal packages( @deriv/shared, @deriv/hooks, ...)
same package directories( /Components, /Containers, ...)

in each category, the sorting should be alphabetically

Reply:
Can we add an eslint rule for this? 
I saw in different PRs we are asking developers to sort the imports, let's make it easier :grin: